docs/advanced_topics/customisation/page_editing_interface.rst

```diff
@@ -57,22 +57,99 @@ Wagtail provides a general-purpose WYSIWYG editor for creating rich text content
 
 However, template output from :class:`~wagtail.wagtailcore.fields.RichTextField` is special and need to be filtered to preserve embedded content. See :ref:`rich-text-filter`.
 
-If you're interested in extending the capabilities of the Wagtail WYSIWYG editor (``hallo.js``), See :ref:`extending_wysiwyg`.
+
+.. _rich_text_features:
+
+Limiting features in a rich text field
+--------------------------------------
+
+By default, the rich text editor provides users with a wide variety of options for text formatting and inserting embedded content such as images. However, we may wish to restrict a rich text field to a more limited set of features - for example:
+
+ * The field might be intended for a short text snippet, such as a summary to be pulled out on index pages, where embedded images or videos would be inappropriate;
+ * When page content is defined using :ref:`StreamField <streamfield>`, elements such as headings, images and videos are usually given their own block types, alongside a rich text block type used for ordinary paragraph text; in this case, allowing headings and images to also exist within the rich text content is redundant (and liable to result in inconsistent designs).
+
+This can be achieved by passing a ``features`` keyword argument to ``RichTextField``, with a list of identifiers for the features you wish to allow:
+
+.. code-block:: python
+
+    body = RichTextField(features=['p', 'h2', 'h3', 'b', 'i', 'link'])
+
+The recognised feature identifiers are as follows (note that add-on modules may add to this list):
+
+ * ``p`` - paragraph text (text entered into a rich text field appears in this mode by default, so omitting this from ``features`` is unlikely to be meaningful)
+ * ``h1``, ``h2``, ``h3``, ``h4``, ``h5``, ``h6`` - heading elements
+ * ``b``, ``i`` - bold / italic text
```

Comment:
On closer inspection, the feature names _are_ `bold` and `italic` - the documentation was wrong. Fixed in ddf93e43a2f3160cc59379abbf9e4483e026b53e... Yep, I think in this case it does make sense to depart from the actual element names, because the _effect_ is to provide bold / italic text, even if it's up in the air which actual elements are used.

Since hallo.js uses `<b>` / `<i>`, and therefore most of the rich text data existing in databases is using those, I think it makes sense to declare `<b>` / `<i>` as the "standard" for rich text in the database, in the interests of not creating extra work for ourselves. That doesn't rule out the possibility of rewriting them to `<strong>` / `<em>` behind the scenes, either for the front-end HTML rendering or for the benefit of editor widgets which use those natively. That's something we don't need to worry about yet, though, since the current PR is only concerned with hallo.js's behaviour.